package.json

```diff
@@ -36,7 +36,7 @@
     "async": "1.5.2",
     "chokidar": "nono/chokidar#unlink-with-awf-cwd",
     "commander": "2.9.0",
-    "cozy-device-sdk": "1.3.1",
+    "cozy-device-sdk": "nono/cozy-device-sdk#build",
```

Comment:
I'm wrong. It's the one for printit that has not yet been merged: https://github.com/cozy/printit/pull/20